.github/workflows/release-latest.yml

```diff
@@ -75,3 +75,41 @@ jobs:
           tag: '${{ env.version }}'
           commit: 'main'
           makeLatest: true
+
+  build_docs:
+    name: Build Docs
+    runs-on: ubuntu-latest
```

Comment:
Add `needs: release_latest` so that the docs are not published if the release fails for some reason.